test/specs/upgrade/pre-upgrade.ts

```diff
@@ -1,7 +1,38 @@
 import assert from 'assert';
+import { getTestString } from 'wdio-mediawiki/Util.js';
 import WikibaseApi from 'wdio-wikibase/wikibase.api.js';
 
 describe( 'Wikibase pre upgrade', function () {
+	it( 'Should be able to create many properties and items', async () => {
+		const numEntities = 100;
+		for ( let i = 0; i < numEntities; i++ ) {
+			const itemLabel = 'T267743-';
+			const propertyValue = 'PropertyExampleStringValue';
+			const propertyId = await WikibaseApi.createProperty( 'string' );
+			const data = {
+				claims: [
+					{
+						mainsnak: {
+							snaktype: 'value',
+							property: propertyId,
+							datavalue: { value: propertyValue, type: 'string' }
+						},
+						type: 'statement',
+						rank: 'normal'
+					}
+				]
+			};
+
+			const itemId = await WikibaseApi.createItem(
+				getTestString( itemLabel ),
+				data
+			);
+
+			assert.strictEqual( itemId.startsWith( 'Q' ), true );
+			assert.strictEqual( propertyId.startsWith( 'P' ), true );
```

Comment:
Is there a reason to use `assert.strictEqual( X, true)` over `assert( X )`?

Reply:
Probably not, but this is not new code, just moved, so I'm going to let it be for now.